### DISCUSSION TITLE
Documentation corrections

### DISCUSSION THREAD

**Discussion by jkost** [microsoft/gctoolkit#191] [General]
Hallo, I start this discussion thread so that others can also contribute on parts of the documentation that don't work or need improvement.

- [GCToolKit introductory sample](https://github.com/microsoft/gctoolkit/blob/main/sample/README.md): to run the sample, `mvnw compile` is not enough; `mvn install` is needed so that the artifacts are created in the local repository and can be found by the next step. Then, one needs to first `cd sample` and then run `mvnw exec:exec` or `mvn exec:exec`.

Btw, is `gctoolkit` the evolution of JClarity's Censum, or not? If yes, how can one access a graphical application to do what Censum used to do?

**Comment by karianna** [comment]
@jkost Please do feel free to submit a PR for the doc improvement.  Yes GCToolkit is the backend of Censum ripped out, enhanced and then Open Sourced.  The front end remains closed source (for now).

**Comment by ghost** [comment]
Hi John (@jkost). I'll be checking in a dev guide today. I just checked in releaser. If you go to releaser.org you'll see that the commit messages follow a format first suggested by Angular. We will also have to add this information to the contributors guide.

**Comment by jkost** [comment]
I can't access releaser.org I 'm afraid. :( However, I can view a Dev's
guide checked-in here
<https://github.com/microsoft/gctoolkit/blob/main/DEVELOPERS_GUIDE.md>
which seems quite useful for someone who wants to start using the library.

Thanks Kirk.

John.

**Reply by kcpeppe** [reply]
Oops, I got autoincorrected. It should be jreleaser.org.
